src/content/fr/2019/seo.md

```diff
@@ -0,0 +1,421 @@
+96,93% des pages mobiles ont réussi [l'audit d'indexation de Lighthouse](https://developers.google.com/web/tools/lighthouse/audits/indexing), ce qui signifie que ces pages ne contenaient pas de directive «noindex». Cependant, cela signifie que 3,07% des pages d'accueil mobiles ont une directive `noindex`, ce qui est préoccupant et signifie que Google n'a pas pu indexer ces pages.
+
+<p class="note">Les sites web inclus dans notre recherche proviennent de {a36 href="./methodology#chrome-ux-report"}{/a36}Chrome UX Report, qui exclut les site web non publiques. Il s'agit d'une source importante de biais, car nous ne sommes pas en mesure d'analyser les sites que Chrome juge non publics. Plus de détails sur notre <a href="./methodology#websites">méthodologie/a>.</p>
+
+#### Canonicalisation
+
+Les balises canoniques sont utilisées pour spécifier les pages en double et leurs alternatives préférées, afin que les moteurs de recherche puissent consolider l'autorité qui pourrait être répartie sur plusieurs pages du groupe sur une seule page principale pour un meilleur classement.
+
+48,34% des pages d'accueil mobiles ont été [détectées](https://developers.google.com/web/tools/lighthouse/audits/canonical) avvoir une balise canonique. Les balises canoniques auto-référencées ne sont pas essentielles et les balises canoniques sont généralement requises pour les pages en double. Les pages d'accueil sont rarement dupliquées ailleurs sur le site, il n'est donc pas surprenant de constater que moins de la moitié des pages ont une balise canonique.
+
+#### robots.txt
+
+L'une des méthodes les plus efficaces pour contrôler l'exploration des moteurs de recherche est le fichier [robots.txt]. Il s'agit d'un fichier qui se trouve sur le domaine racine d'un site web et spécifie quelles URL et chemins d'URL doivent être interdits à l'exploration par les moteurs de recherche.
+
+Il était intéressant de constater que seulement 72,16% des sites mobiles ont un «robots.txt» valide, [selon Lighthouse](https://developers.google.com/web/tools/lighthouse/audits/robots). The Les principaux problèmes que nous avons constatés sont répartis entre 22% des sites n'ayant aucun fichier `robots.txt` et ~ 6% servant un fichier` robots.txt` non valide, et échouent ainsi l'audit. Bien qu'il existe de nombreuses raisons valables de ne pas avoir de fichier `robots.txt`, comme avoir un petit site web qui n'a pas [de soucis de budget de crawl](https://webmasters.googleblog.com/2017/01/what-crawl-budget-means-for-googlebot.html), avoir un `robots.txt` invalide peut être problématique surtout avec le Mobile First Index.
```

Comment:
The Les/Les

Reply:
quelles URL/quelles URL**s** ?

Reply:
échouent ainsi l'audit/échouent ainsi à l'audit/sont en échec à l'audit ?

Reply:
OK pour point 1 et 3. URL reste au singulier